sdk/python/tests/compiler/compiler_tests.py

```diff
@@ -290,6 +290,10 @@ def test_py_compile_basic(self):
     """Test basic sequential pipeline."""
     self._test_py_compile_zip('basic')
 
+  def test_py_compile_with_init_container(self):
```

Comment:
Can you please make a test that tests the behavior of the new API feature (as opposed to just checking that nothing has changed)? Good examples are `test_set_display_name`, `test_op_transformers`.

Reply:
done